bin/release-plugin-to-dot-org.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+CURRENT_PATH=`pwd`
+
+# Pull down the SVN repository.
+echo "Pulling down the SVN repository for woocommerce-admin"
+SVN_WOOCOMMERCE_ADMIN_PATH=/tmp/woocommerce/svn-woocommerce-admin
+svn co https://plugins.svn.wordpress.org/woocommerce-admin/ $SVN_WOOCOMMERCE_ADMIN_PATH
+cd  $SVN_WOOCOMMERCE_ADMIN_PATH
+
+# Get the tagged version to release.
+echo "Please enter the version number to release to woocommerce.org, for example, 1.0.0: "
+read -r VERSION
+
+# Empty trunk/.
+rm -rf trunk
+mkdir trunk
+
+# Download and unzip the plugin into trunk/.
+echo "Downloading and unzipping the plugin"
+PLUGIN_URL=https://github.com/woocommerce/woocommerce-admin/releases/download/v${VERSION}-plugin/woocommerce-admin.zip
+curl -Lo woocommerce-admin.zip $PLUGIN_URL
+unzip woocommerce-admin.zip -d trunk
+rm woocommerce-admin.zip
+
+# Add files in trunk/ to SVN.
+cd trunk
+svn add --force .
```

Comment:
The script follows the existing deploy instructions. Are you seeing previous chunks because they're in your git repo's `dist` folder? There's a step in the deploy instructions to clone a fresh copy of the git repo before generating a dist zip.

Reply:
The previous chunks remain in the SVN repo - they need to be removed from SVN, not just deleted locally.

Reply:
Oh nasty. I still wonder why this isn't mentioned in the [deploy instructions](https://teamisotope.wordpress.com/field-notes/#woocommerce-admin-release) - it just says to replace the contents of `trunk/` with the plugin zip contents.

I'll try removing them from SVN as well. Thanks.

Reply:
It might not be an issue anymore because the chunkname PR landed.